pkgs/by-name/ne/netop/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, libpcap, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "netop";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "ZingerLittleBee";
+    repo = "netop";
+    rev = "v${version}";
+    hash = "sha256-Rnp2VNAi8BNbKqkGFoYUb4C5db5BS1P1cqpWlroTmdQ=";
+  };
+
+  LIBPCAP_LIBDIR = lib.makeLibraryPath [ libpcap ];
+  LIBPCAP_VER = libpcap.version;
+
+  cargoSha256 = "5vbv4w17DdaTKuF3vQOfv74I8hp2Zpsp40ZlF08qWlc=";
```

Comment:
Should it be `vendorHash` ?

Reply:
I don't think so?

```
error: cargoSha256, cargoHash, cargoVendorDir, or cargoLock must be set
```

Reply:
Yes sorry it must be `cargoHash` !

Reply:
I guess I can, but I'd have to prefix it with sha256- then. Out of curiosity, why is this preferred? According to https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/rust.section.md both forms are accepted to nixpkgs.

Reply:
Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.